calico-enterprise_versioned_docs/version-3.16/_includes/release-notes/_v3.16.1-release-notes.mdx

```diff
@@ -0,0 +1,60 @@
+
+{{prodname}} has added egress gateway support for Microsoft Azure and AKS.
+egress gateways allow you to identify the namespaces associated with egress traffic outside your cluster.
+
+For more information, see [Configure egress gateways, Azure](../../networking/egress/egress-gateway-azure.mdx).
+
+#### Operator-managed deployments of egress gateways
+
+{{prodname}} now includes Operator-managed deployments of egress gateways.
+This reduces operational overhead and eliminates additional steps required during upgrades.
+
+For more information, see [Egress gateways](../networking/egress/).
+
+#### UI for workload-based web application firewalls
+
+{{prodname}} includes a new UI that can be used to enable and configure a workload-based web application firewall.
+
+For more information, see [Workload-based web application firewall](../../threat/web-application-firewall.mdx#manager-ui).
+
+#### Service Graph performance optimizations
+
+This release of {{prodname}} includes several optimizations to improve the performance of Service Graph for clusters with larger numbers of namespaces.
+
+#### Revised installation steps for Rancher RKE2
+
+{{prodname}} has revised the installation steps for Rancher RKE2 to accommodate options available in the most recent versions of that platform.
+
+<!--The following anchor text and link need to be updated to /rke2.mdx after updates are incorporated-->
+For more information, see [Rancher Manager](../../getting-started/install-on-clusters/rancher-ui.mdx).
+
+#### Improvements to Envoy to accommodate advanced ingress controllers
+
+{{prodname}} includes improvements to its Envoy deployment to enable customers to use this feature in clusters with ingress controllers that perform advanced load balancing.
+
+For more information, see [Workload-based web application firewall](../../threat/web-application-firewall.mdx).
+
+#### Support for multiple external networks
+
+{{prodname}} includes networking options that allows pods from different namespaces to egress onto different external networks (Virtual Routing Functions or VRFs) that may have overlapping IPs with each other.
+This also enables administrators to configure which service IPs are advertised for a network by the service IP advertisement feature.
+
+For more information, see [Configure egress traffic to multiple external networks](../../networking/egress/external-network.mdx).
+
+#### Improved product security with more restrictive `SecurityContext`
+
+{{prodname}} has updated its components to apply a more restrictive `SecurityContext` where applicable.
+This includes the following:
+* Non-root context whenever possible and added drop ALL `Capabilities`
+* Root context and privilege escalation are used only when necessary
+* `SeccompProfile` is set to `RuntimeDefault`
+
+## Known issues
+
+* L7 log collection fails to deploy on CIS hardened clusters. As a result, some cards in the manager UI dashboard will not display any metrics.
```

Comment:
This issue is resolved and can be removed.

Reply:
Deleted. Thanks!